src/ConfigManager.h

```diff
@@ -73,9 +72,13 @@ class ConfigParameter : public BaseParameter {
 
   ParameterMode getMode() { return this->mode; }
 
+  void update(T value) {
```

Comment:
I like it 👍 It gives me an idea for the future to move JSON out of ConfigManager.

Reply:
That would be awesome, a pure `update` hook of the data